### PR TITLE
Np 1387 remove mapper usage

### PR DESCRIPTION
**Pull request by axthosarouris**
Removed DynamoDb mapper because it cannot handle well enums that will be added later. 

15 classes have just only one empty line removed.

### CODE REVIEW

File: user-access-commons/src/test/java/no/unit/nva/database/DatabaseServiceTest.java [hunk 1]
```diff
@@ -240,6 +244,23 @@ public void listUsersByInstitutionReturnsEmptyListWhenThereAreNoUsersForSpecifie
         assertThat(queryResult, is(empty()));
     }
 
+    @Test
+    void createTableThrowsExceptionWhenDynamoClientIsNull() {
```
Comment (axthosarouris) [inline user-access-commons/src/test/java/no/unit/nva/database/DatabaseServiceTest.java:248]: tests moved from deleted classes `DatabaseServiceWithTableNameOverride` and `DatabaseServiceWithTableNameOverrideTest`

File: user-access-commons/src/main/java/no/unit/nva/database/DatabaseServiceImpl.java [hunk 2]
```diff
@@ -169,17 +192,33 @@ private static boolean isInvalid(Validable roleDto) {
         return isNull(roleDto) || !roleDto.isValid();
     }
 
+    private static RuntimeException logErrorWithDynamoClientAndThrowException(Failure<AmazonDynamoDB> failure) {
```
Comment (norrs) [inline user-access-commons/src/main/java/no/unit/nva/database/DatabaseServiceImpl.java:195]: Implementation here does return a RuntimeException, but throws it.  Make void?

**Review by norrs — CHANGES_REQUESTED** [review]

**Review by norrs — APPROVED** [review]
nitpick the one about the runtime exception, do a quick fix and LGTM.

**Review by janbj — APPROVED** [review]

File: user-access-commons/src/main/java/no/unit/nva/database/DatabaseServiceImpl.java (hunk 2 above)
Comment (axthosarouris) [reply user-access-commons/src/main/java/no/unit/nva/database/DatabaseServiceImpl.java:195]: nice catch

**Pull request merged**